practice_two/src/styles/sections/management.css

```diff
@@ -0,0 +1,32 @@
+.management {
+	display: grid;
```

Comment:
I don't see any properties relate to the grid layout you set? Anything wrong here?

Reply:
Sorry for the confusion here, i fixed it